byron/ledger/formal-spec/properties.tex

```diff
@@ -156,3 +156,76 @@ \subsection{UTxO Properties}
   $$
 
 \end{property}
+
+\subsection{PVBUMP STS Properties}
```

Comment:
It'd be nice to be consistent with the name of the other sections, we either make all `$LABEL STS Properties` or `$LABEL Properties`. I'd prefer the latter, since it is inside a section named "Transition System Properties".

Also we might want to add this into a sub-sub section, inside a section named "Update Properties" or something like that.

Reply:
I addressed both of your comments. Thanks!